packages/ckeditor5-dev-env/lib/index.js

```diff
@@ -5,7 +5,13 @@
 
 'use strict';
 
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
 const executeOnDependencies = require( './release-tools/utils/executeondependencies' );
+const packagesToRelease = require( './release-tools/utils/packagestorelease' );
+const validator = require( './release-tools/utils/releasevalidator' );
+const cli = require( './release-tools/utils/cli' );
+
+const BREAK_RELEASE_MESSAGE = 'Creating release has been aborted.';
```

Comment:
Process of creating a release is also called "releasing" ;P

"Releasing has been aborted due to errors." would be better.

But what would be even better is if the error which is thrown had a proper message – i.e. containing a reason.

Instead of logging and throwing just throw and log the error in a `catch()`.